### ISSUE TITLE
Support creator-side zap splits

### ISSUE THREAD

**Issue by staab** [coracle-social/coracle#419]
Configurable either on the platform level, on the user level, or on the note publish options level.

**Comment by ticruz38** [comment]
Just to make sure I understand it correctly

platform level: Configurable via environment variables?
user level: in the UserSetting view as we do now
note publish: Just before zapping, in the modal?

**Comment by staab** [comment]
Yes, but this is actually for zap _split_, not zap amount. So when you're creating a kind 1, allow the user to split zaps with other people in the options modal.

On second though though, I think this might not be worth supporting. I don't think it's ever actually been requested.
